### ISSUE TITLE
Weird action of Articulate

### ISSUE THREAD

**Issue by bkurowski** [samtecspg/articulate#551]
### Issues
Hi guys, I am having issue here that i never met before. And not sure how it is even possible.

This is the scenario:

![image](https://user-images.githubusercontent.com/40358189/52337804-9e8e4000-2a00-11e9-8e5e-41c35a83fb57.png)

I am asking for weather, the webhook, slots and agent response are created exactly the same as in my previous version of Articulate 0.13. Screenshots visible below.

***User Says***
![image](https://user-images.githubusercontent.com/40358189/52338017-22e0c300-2a01-11e9-961d-3ee48d325e93.png)

***City Slots***
![image](https://user-images.githubusercontent.com/40358189/52338070-3f7cfb00-2a01-11e9-8d60-1a0cd56056b3.png)

***Units Slots***
![image](https://user-images.githubusercontent.com/40358189/52338102-57ed1580-2a01-11e9-8728-2e9c8a3fe8d0.png)

***Webhook***
Of course the link is complete, for good explanation the link is not visible fully
![image](https://user-images.githubusercontent.com/40358189/52338170-91258580-2a01-11e9-9e34-a1d3c757d5d2.png)

***Agent Response***
![image](https://user-images.githubusercontent.com/40358189/52338201-a3072880-2a01-11e9-8980-b9b73ed1b80d.png)

I am lost on why it is giving me weather question on a Greeting where as weather action is not added to a greeting and why the slots are not taking in the city and unit and it is not putting those details into the webhook link provided.

Cheers for answers guys !! 

All the best .

**Comment by wrathagom** [comment]
@bkurowski click on the see source button and inspect the context, you'll see that there is an actionQueue. When you said hi a final time it recognized that as a new action, so it responded with `Welcome User`, but then it continued the conversation where it left off trying to fill the city slot of your weather action.

**Comment by wrathagom** [comment]
If you wan to start the conversation over again click `Clear All` at the top of the chat panel.

**Comment by bkurowski** [comment]
Ok thats understandable, i could not test that on custom chat as i can not get the connection to my virtual machine as i explained just in Slack platform. So how does it look like in custom chat ? If this situation happens ? I need to clear the chat too for that ? Or somehow break out from context ? 
Cheers for fast answer !

**Comment by bkurowski** [comment]
And another thing what could be the reason that my inputs ale not passed through ? The request can not be completed.

**Comment by bkurowski** [comment]
Ok i think i see my problem now, Sorry for flooding the page here.. 

The response can not be exactly the same format as in old UI, can you tell me if there are any changes in handlebars for the new UI ? 

Because the agent response that is visible for you above works in old UI and does not in new UI.

Cheers and sorry again for flooding

**Comment by wrathagom** [comment]
no handlebar changes that I am aware of. The error that you showed just suggest that the value of the slot wasn't set meaning Articulate did not successfully parse the city response. You can look at the `See Source` button and Parse Results to see if it Dublin was recognized as a keyword.

The action will be cleared from the queue once fulfilled, you can also use the API to clear it if need be.

**Comment by bkurowski** [comment]
I see your point, i found the error, that was my fault. Cheers for your time and information. I need to pay more attention to the new option "See source" 

Thanks @wrathagom !

**Comment by bkurowski** [comment]
Here is a good example of queue being fulfilled and still have the same situation. And i am wondering will the  user have to clear that everytime ? Either using API  or manually ?  

![image](https://user-images.githubusercontent.com/40358189/52340993-47409d80-2a09-11e9-835a-6de9d586d7ac.png)

**Comment by wrathagom** [comment]
can you provide an export of your agent? Looks like maybe the action got triggered twice.

**Comment by bkurowski** [comment]
```
{
  "agentName": "Ericsson",
  "description": "Answers Questions",
  "language": "en",
  "timezone": "UTC",
  "useWebhook": false,
  "usePostFormat": false,
  "multiCategory": true,
  "categoryClassifierThreshold": 0.5,
  "fallbackAction": "Default Fallback Action",
  "status": "Ready",
  "lastTraining": "2019-02-05T18:52:03Z",
  "extraTrainingData": false,
  "enableModelsPerCategory": true,
  "model": "59900a43-f569-8014-6263-88428db7a85a",
  "parameters": {},
  "settings": {
    "rasaURL": "http://rasa:5000",
    "ducklingURL": "http://duckling:8000",
    "ducklingDimension": [
      "amount-of-money",
      "distance",
      "duration",
      "email",
      "number",
      "ordinal",
      "phone-number",
      "quantity",
      "temperature",
      "time",
      "url",
      "volume"
    ],
    "spacyPretrainedEntities": [
      "PERSON",
      "NORP",
      "FAC",
      "ORG",
      "GPE",
      "LOC",
      "PRODUCT",
      "EVENT",
      "WORK_OF_ART",
      "LAW",
      "LANGUAGE",
      "DATE",
      "TIME",
      "PERCENT",
      "MONEY",
      "QUANTITY",
      "ORDINAL",
      "CARDINAL"
    ],
    "categoryClassifierPipeline": [
      {
        "name": "intent_featurizer_count_vectors"
      },
      {
        "name": "intent_classifier_tensorflow_embedding",
        "intent_tokenization_flag": true,
        "intent_split_symbol": "+"
      },
      {
        "name": "nlp_spacy"
      },
      {
        "name": "tokenizer_spacy"
      },
      {
        "name": "ner_crf"
      },
      {
        "name": "ner_synonyms"
      },
      {
        "name": "ner_spacy"
      }
    ],
    "sayingClassifierPipeline": [
      {
        "name": "intent_featurizer_count_vectors"
      },
      {
        "name": "intent_classifier_tensorflow_embedding",
        "intent_tokenization_flag": true,
        "intent_split_symbol": "+"
      },
      {
        "name": "nlp_spacy"
      },
      {
        "name": "tokenizer_spacy"
      },
      {
        "name": "ner_crf"
      },
      {
        "name": "ner_synonyms"
      },
      {
        "name": "ner_spacy"
      }
    ],
    "keywordClassifierPipeline": [
      {
        "name": "nlp_spacy"
      },
      {
        "name": "tokenizer_spacy"
      },
      {
        "name": "ner_crf"
      },
      {
        "name": "ner_synonyms"
      },
      {
        "name": "ner_spacy"
      }
    ],
    "sayingsPageSize": 50,
    "keywordsPageSize": 5,
    "modifierSayingsPageSize": 5,
    "reviewPageSize": 5
  },
  "categoryRecognizer": true,
  "modifiersRecognizer": false,
  "modifiersRecognizerJustER": "",
  "creationDate": "1549372413000",
  "modificationDate": "1549393169000",
  "actions": [
    {
      "actionName": "Default Fallback Action",
      "slots": [],
      "responses": [
        {
          "textResponse": "Sorry can you rephrase that?",
          "actions": []
        },
        {
          "textResponse": "I'm still learning to speak with humans. What you mean?",
          "actions": []
        }
      ],
      "useWebhook": false,
      "usePostFormat": false,
      "creationDate": "1549372413000",
      "modificationDate": "1549372413000"
    },
    {
      "actionName": "Greet",
      "slots": [],
      "responses": [
        {
          "textResponse": "Welcome user.",
          "actions": []
        }
      ],
      "useWebhook": false,
      "usePostFormat": false,
      "creationDate": "1549372413000",
      "modificationDate": "1549372413000"
    },
    {
      "actionName": "DateAction",
      "slots": [],
      "responses": [
        {
          "textResponse": "Today is {{webhookResponse.dayofweekName}}. {{ordinalize webhookResponse.day}} of {{webhookResponse.monthName}} - {{webhookResponse.year}}",
          "actions": []
        }
      ],
      "useWebhook": true,
      "usePostFormat": false,
      "creationDate": "1549372413000",
      "modificationDate": "1549372413000",
      "webhook": {
        "webhookUrl": "https://script.google.com/macros/s/AKfycbyd5AcbAnWi2Yn0xhFRbyzS4qMq1VucMVgVvhul5XqS9HkAyJY/exec",
        "webhookVerb": "GET",
        "webhookPayloadType": "None",
        "webhookPayload": "",
        "webhookHeaders": [],
        "webhookUser": "",
        "webhookPassword": "",
        "creationDate": "1549372413000",
        "modificationDate": "1549372413000"
      }
    },
    {
      "actionName": "TimeAction",
      "slots": [],
      "responses": [
        {
          "textResponse": "It is :.. {{webhookResponse.hours}}:{{webhookResponse.minutes}}",
          "actions": []
        }
      ],
      "useWebhook": true,
      "usePostFormat": false,
      "creationDate": "1549372413000",
      "modificationDate": "1549372413000",
      "webhook": {
        "webhookUrl": "https://script.google.com/macros/s/AKfycbyd5AcbAnWi2Yn0xhFRbyzS4qMq1VucMVgVvhul5XqS9HkAyJY/exec",
        "webhookVerb": "GET",
        "webhookPayloadType": "None",
        "webhookPayload": "",
        "webhookHeaders": [],
        "webhookUser": "",
        "webhookPassword": "",
        "creationDate": "1549372413000",
        "modificationDate": "1549372413000"
      }
    },
    {
      "actionName": "SlotsAction",
      "slots": [],
      "responses": [
        {
          "textResponse": "This is what I am expecting :.. {{slots.City.original}}",
          "actions": []
        }
      ],
      "useWebhook": false,
      "usePostFormat": false,
      "creationDate": "1549372413000",
      "modificationDate": "1549372413000"
    },
    {
      "actionName": "Net_Issue_Action",
      "slots": [],
      "responses": [
        {
          "textResponse": "Key : {{webhookResponse.[0].key}}, Value : {{webhookResponse.[0].value}}, Key : {{webhookResponse.[1].key}}, Value : {{webhookResponse.[1].value}}, Key : {{webhookResponse.[2].key}}, Value : {{webhookResponse.[2].value}}",
          "actions": []
        }
      ],
      "useWebhook": true,
      "usePostFormat": false,
      "creationDate": "1549373104000",
      "modificationDate": "1549373104000",
      "webhook": {
        "webhookUrl": "http://10.45.16.205:8080/incident-management-core-app/rest/knowledge-base/dimension-keys/SPECIFIC_PROBLEM/occurrence-count",
        "webhookVerb": "GET",
        "webhookPayloadType": "None",
        "webhookPayload": "",
        "webhookHeaders": [],
        "webhookUser": "",
        "webhookPassword": "",
        "creationDate": "1549373104000",
        "modificationDate": "1549373104000"
      }
    },
    {
      "actionName": "Leave_Conversation",
      "slots": [],
      "responses": [
        {
          "textResponse": "Bye, have a nice day !",
          "actions": []
        }
      ],
      "useWebhook": false,
      "usePostFormat": false,
      "creationDate": "1549373329000",
      "modificationDate": "1549373329000"
    },
    {
      "actionName": "Anomaly_Action",
      "slots": [
        {
          "slotName": "Cell",
          "uiColor": "#194d33",
          "keyword": "Cell",
          "keywordId": 0,
          "isList": false,
          "isRequired": true,
          "textPrompts": [
            "Please specify cell name:"
          ]
        }
      ],
      "responses": [
        {
          "textResponse": "{{webhookResponse}}",
          "actions": []
        }
      ],
      "useWebhook": true,
      "usePostFormat": false,
      "creationDate": "1549373849000",
      "modificationDate": "1549393169000",
      "webhook": {
        "webhookUrl": "http://10.45.16.205:1880/anomalyDet0",
        "webhookVerb": "POST",
        "webhookPayloadType": "JSON",
        "webhookPayload": "{\r\n    \"dimensionkey\": [\r\n        {{{JSONstringify slots.Cell.original}}}\r\n    ],\r\n    \"Mannual\": false,\r\n    \"incidentid\": \"\",\r\n    \"incidentTime\": \"1511450100000\",\r\n    \"Injcounter\": [\r\n        \"pmActiveUeUlMax\",\"pmRaAttCfra\",\r\n        \"pmRrcConnEstabSuccMos\" \r\n    ],\r\n    \"isbot\" : true\r\n}",
        "webhookHeaders": [],
        "webhookUser": "",
        "webhookPassword": "",
        "creationDate": "1549373849000",
        "modificationDate": "1549393169000"
      }
    },
    {
      "actionName": "SlotActionTest",
      "slots": [
        {
          "slotName": "City",
          "uiColor": "#0288d1",
          "keyword": "City",
          "keywordId": 0,
          "isList": false,
          "isRequired": true,
          "textPrompts": [
            "Specify city name please"
          ]
        }
      ],
      "responses": [
        {
          "textResponse": "This is what is expected {{slots.City.original}}",
          "actions": []
        }
      ],
      "useWebhook": false,
      "usePostFormat": false,
      "creationDate": "1549383103000",
      "modificationDate": "1549383103000"
    },
    {
      "actionName": "NetworkStatusAction",
      "slots": [],
      "responses": [
        {
          "textResponse": "There are/is {{webhookResponse.[Open Incidents]}} open incident/s. You can view those here: http://10.45.16.205:8080/#incidentclassification/incidentManagement",
          "actions": []
        }
      ],
      "useWebhook": true,
      "usePostFormat": false,
      "creationDate": "1549385698000",
      "modificationDate": "1549385698000",
      "webhook": {
        "webhookUrl": "http://10.45.16.205:8080/associationmapper-app/rest/incidents/stats",
        "webhookVerb": "GET",
        "webhookPayloadType": "None",
        "webhookPayload": "",
        "webhookHeaders": [],
        "webhookUser": "",
        "webhookPassword": "",
        "creationDate": "1549385698000",
        "modificationDate": "1549385698000"
      }
    },
    {
      "actionName": "WeatherAction",
      "slots": [
        {
          "slotName": "City",
          "uiColor": "#0288d1",
          "keyword": "City",
          "keywordId": 0,
          "isList": false,
          "isRequired": true,
          "textPrompts": [
            "Please specify city name:"
          ]
        },
        {
          "slotName": "units",
          "uiColor": "#f57f17",
          "keyword": "units",
          "keywordId": 0,
          "isList": false,
          "isRequired": true,
          "textPrompts": [
            "Please specify if you would like to view details in metric or imperial units."
          ]
        }
      ],
      "responses": [
        {
          "textResponse": "City Name - {{webhookResponse.name}}, Description - {{webhookResponse.weather.[0].description}}, Pressure - {{webhookResponse.main.pressure}} hPa, Humidity - {{webhookResponse.main.humidity}} %, Cloudiness - {{webhookResponse.clouds.all}} %, {{#contains slots.units.value \"metric\"}} Temperature - {{webhookResponse.main.temp}}°C, Wind Speed - {{webhookResponse.wind.speed}} km/h {{else}} Temperature - {{webhookResponse.main.temp}}°F, Wind Speed - {{webhookResponse.wind.speed}} mp/h {{/contains}}",
          "actions": []
        }
      ],
      "useWebhook": true,
      "usePostFormat": false,
      "creationDate": "1549386694000",
      "modificationDate": "1549391525000",
      "webhook": {
        "webhookUrl": "http://api.openweathermap.org/data/2.5/weather?q={{slots.City.original}}&appid=93faac6a336079bc64d5237fbd6f40dc&units={{slots.units.value}}",
        "webhookVerb": "GET",
        "webhookPayloadType": "None",
        "webhookPayload": "",
        "webhookHeaders": [],
        "webhookUser": "",
        "webhookPassword": "",
        "creationDate": "1549386695000",
        "modificationDate": "1549391525000"
      }
    }
  ],
  "categories": [
    {
      "categoryName": "Greetings",
      "enabled": true,
      "actionThreshold": 0.5,
      "status": "Ready",
      "lastTraining": "2019-02-05T13:30:06Z",
      "model": "Greetings_10197ff1-7779-da7f-335a-c4714c2cd607",
      "extraTrainingData": false,
      "parameters": {},
      "creationDate": "1549372413000",
      "modificationDate": "1549373406000",
      "sayings": [
        {
          "userSays": "Hi",
          "keywords": [],
          "actions": [
            "Greet"
          ],
          "creationDate": "1549372413000",
          "modificationDate": "1549372413000"
        },
        {
          "userSays": "Hey",
          "keywords": [],
          "actions": [
            "Greet"
          ],
          "creationDate": "1549372413000",
          "modificationDate": "1549372413000"
        },
        {
          "userSays": "Hello",
          "keywords": [],
          "actions": [
            "Greet"
          ],
          "creationDate": "1549372413000",
          "modificationDate": "1549372413000"
        },
        {
          "userSays": "Greetings",
          "keywords": [],
          "actions": [
            "Greet"
          ],
          "creationDate": "1549372413000",
          "modificationDate": "1549372413000"
        },
        {
          "userSays": "Bye",
          "keywords": [],
          "actions": [
            "Leave_Conversation"
          ],
          "creationDate": "1549373282000",
          "modificationDate": "1549373329000"
        },
        {
          "userSays": "Goodbye",
          "keywords": [],
          "actions": [
            "Leave_Conversation"
          ],
          "creationDate": "1549373285000",
          "modificationDate": "1549373344000"
        },
        {
          "userSays": "see you",
          "keywords": [],
          "actions": [
            "Leave_Conversation"
          ],
          "creationDate": "1549373288000",
          "modificationDate": "1549373341000"
        },
        {
          "userSays": "c ya",
          "keywords": [],
          "actions": [
            "Leave_Conversation"
          ],
          "creationDate": "1549373290000",
          "modificationDate": "1549373339000"
        }
      ]
    },
    {
      "categoryName": "Time",
      "enabled": true,
      "actionThreshold": 0.5,
      "status": "Ready",
      "lastTraining": "2019-02-05T13:15:29Z",
      "model": "Time_just_er_87f752d1-3494-ec3d-26d2-b372c54400f7",
      "extraTrainingData": false,
      "parameters": {},
      "creationDate": "1549372413000",
      "modificationDate": "1549372529000",
      "sayings": [
        {
          "userSays": "what is the current time",
          "keywords": [],
          "actions": [
            "TimeAction"
          ],
          "creationDate": "1549372413000",
          "modificationDate": "1549372413000"
        },
        {
          "userSays": "whats the time",
          "keywords": [],
          "actions": [
            "TimeAction"
          ],
          "creationDate": "1549372413000",
          "modificationDate": "1549372413000"
        }
      ]
    },
    {
      "categoryName": "Date",
      "enabled": true,
      "actionThreshold": 0.5,
      "status": "Ready",
      "lastTraining": "2019-02-05T13:16:26Z",
      "model": "Date_just_er_3fba4e27-0add-d168-cd21-b29bc4bea6b7",
      "extraTrainingData": false,
      "parameters": {},
      "creationDate": "1549372413000",
      "modificationDate": "1549372586000",
      "sayings": [
        {
          "userSays": "What is todays date",
          "keywords": [],
          "actions": [
            "DateAction"
          ],
          "creationDate": "1549372413000",
          "modificationDate": "1549372413000"
        },
        {
          "userSays": "current date",
          "keywords": [],
          "actions": [
            "DateAction"
          ],
          "creationDate": "1549372413000",
          "modificationDate": "1549372413000"
        }
      ]
    },
    {
      "categoryName": "Network_Issues",
      "enabled": true,
      "actionThreshold": 0.5,
      "status": "Ready",
      "lastTraining": "2019-02-05T13:26:16Z",
      "model": "Network_Issues_just_er_f69e46b4-4e3c-d05f-7431-dde22831d877",
      "extraTrainingData": false,
      "parameters": {},
      "creationDate": "1549372956000",
      "modificationDate": "1549373176000",
      "sayings": [
        {
          "userSays": "what are the biggest problems on my network",
          "keywords": [],
          "actions": [
            "Net_Issue_Action"
          ],
          "creationDate": "1549372970000",
          "modificationDate": "1549373117000"
        },
        {
          "userSays": "are there any issues in my network",
          "keywords": [],
          "actions": [
            "Net_Issue_Action"
          ],
          "creationDate": "1549372989000",
          "modificationDate": "1549373104000"
        }
      ]
    },
    {
      "categoryName": "Anomaly_Det",
      "enabled": true,
      "actionThreshold": 0.5,
      "status": "Ready",
      "lastTraining": "2019-02-05T18:52:03Z",
      "model": "Anomaly_Det_just_er_04d09d9c-00b3-d571-9734-a8a1fc75d6d1",
      "extraTrainingData": false,
      "parameters": {},
      "creationDate": "1549373517000",
      "modificationDate": "1549392723000",
      "sayings": [
        {
          "userSays": "are there any anomalies on cell L2B_0205B",
          "keywords": [
            {
              "value": "L2B_0205B",
              "keyword": "Cell",
              "start": 32,
              "end": 41,
              "keywordId": 2
            }
          ],
          "actions": [
            "Anomaly_Action"
          ],
          "creationDate": "1549373540000",
          "modificationDate": "1549373859000"
        },
        {
          "userSays": "list all anomalies for cell L2B_0209A",
          "keywords": [
            {
              "value": "L2B_0209A",
              "keyword": "Cell",
              "start": 28,
              "end": 37,
              "keywordId": 2
            }
          ],
          "actions": [
            "Anomaly_Action"
          ],
          "creationDate": "1549373560000",
          "modificationDate": "1549373857000"
        },
        {
          "userSays": "show anomalies for L2B_2167A",
          "keywords": [
            {
              "value": "L2B_2167A",
              "keyword": "Cell",
              "start": 19,
              "end": 28,
              "keywordId": 2
            }
          ],
          "actions": [
            "Anomaly_Action"
          ],
          "creationDate": "1549373572000",
          "modificationDate": "1549373849000"
        },
        {
          "userSays": "L2B_0202B",
          "keywords": [
            {
              "value": "L2B_0202B",
              "keyword": "Cell",
              "start": 0,
              "end": 9,
              "keywordId": 2
            }
          ],
          "actions": [],
          "creationDate": "1549374148000",
          "modificationDate": "1549392415000"
        },
        {
          "userSays": "L2B_0206B",
          "keywords": [
            {
              "value": "L2B_0206B",
              "keyword": "Cell",
              "start": 0,
              "end": 9,
              "keywordId": 2
            }
          ],
          "actions": [],
          "creationDate": "1549374160000",
          "modificationDate": "1549392414000"
        },
        {
          "userSays": "L2B_0223C",
          "keywords": [
            {
              "value": "L2B_0223C",
              "keyword": "Cell",
              "start": 0,
              "end": 9,
              "keywordId": 2
            }
          ],
          "actions": [],
          "creationDate": "1549374180000",
          "modificationDate": "1549392412000"
        },
        {
          "userSays": "show anomalies",
          "keywords": [],
          "actions": [
            "Anomaly_Action"
          ],
          "creationDate": "1549392646000",
          "modificationDate": "1549392658000"
        }
      ]
    },
    {
      "categoryName": "Test",
      "enabled": true,
      "actionThreshold": 0.5,
      "status": "Ready",
      "lastTraining": "2019-02-05T17:12:39Z",
      "model": "Test_just_er_3bd149e3-c20c-fc10-2a61-cda161903574",
      "extraTrainingData": false,
      "parameters": {},
      "creationDate": "1549382978000",
      "modificationDate": "1549386759000",
      "sayings": [
        {
          "userSays": "test slot for Barcelona",
          "keywords": [
            {
              "value": "Barcelona",
              "keyword": "City",
              "start": 14,
              "end": 23,
              "keywordId": 1
            }
          ],
          "actions": [
            "SlotActionTest"
          ],
          "creationDate": "1549382995000",
          "modificationDate": "1549383131000"
        },
        {
          "userSays": "slots test for Madrid",
          "keywords": [
            {
              "value": "Madrid",
              "keyword": "City",
              "start": 15,
              "end": 21,
              "keywordId": 1
            }
          ],
          "actions": [
            "SlotActionTest"
          ],
          "creationDate": "1549383002000",
          "modificationDate": "1549383127000"
        },
        {
          "userSays": "test for Seville",
          "keywords": [
            {
              "value": "Seville",
              "keyword": "City",
              "start": 9,
              "end": 16,
              "keywordId": 1
            }
          ],
          "actions": [
            "SlotActionTest"
          ],
          "creationDate": "1549383019000",
          "modificationDate": "1549383123000"
        }
      ]
    },
    {
      "categoryName": "Network_Status",
      "enabled": true,
      "actionThreshold": 0.5,
      "status": "Ready",
      "lastTraining": "2019-02-05T16:56:22Z",
      "model": "Network_Status_just_er_50fc5c1d-f2e9-c86d-b076-d70cebf8a278",
      "extraTrainingData": false,
      "parameters": {},
      "creationDate": "1549385611000",
      "modificationDate": "1549385782000",
      "sayings": [
        {
          "userSays": "how is my network",
          "keywords": [],
          "actions": [
            "NetworkStatusAction"
          ],
          "creationDate": "1549385627000",
          "modificationDate": "1549385710000"
        },
        {
          "userSays": "show me status of my network",
          "keywords": [],
          "actions": [
            "NetworkStatusAction"
          ],
          "creationDate": "1549385640000",
          "modificationDate": "1549385698000"
        }
      ]
    },
    {
      "categoryName": "Weather",
      "enabled": true,
      "actionThreshold": 0.5,
      "status": "Ready",
      "lastTraining": "2019-02-05T18:16:28Z",
      "model": "Weather_just_er_43296315-f403-d123-c0a9-d632a1b62bad",
      "extraTrainingData": false,
      "parameters": {},
      "creationDate": "1549386011000",
      "modificationDate": "1549390588000",
      "sayings": [
        {
          "userSays": "what is the weather like in Dublin using metric units",
          "keywords": [
            {
              "value": "Dublin",
              "keyword": "City",
              "start": 28,
              "end": 34,
              "keywordId": 1
            },
            {
              "value": "metric",
              "keyword": "units",
              "start": 41,
              "end": 47,
              "keywordId": 3
            }
          ],
          "actions": [
            "WeatherAction"
          ],
          "creationDate": "1549386039000",
          "modificationDate": "1549387101000"
        },
        {
          "userSays": "show me weather",
          "keywords": [],
          "actions": [
            "WeatherAction"
          ],
          "creationDate": "1549386061000",
          "modificationDate": "1549387098000"
        },
        {
          "userSays": "show me weather in Belfast with imperial units",
          "keywords": [
            {
              "value": "Belfast",
              "keyword": "City",
              "start": 19,
              "end": 26,
              "keywordId": 1
            },
            {
              "value": "imperial",
              "keyword": "units",
              "start": 32,
              "end": 40,
              "keywordId": 3
            }
          ],
          "actions": [
            "WeatherAction"
          ],
          "creationDate": "1549386469000",
          "modificationDate": "1549386695000"
        },
        {
          "userSays": "show me weather for Galway using metric units",
          "keywords": [
            {
              "value": "Galway",
              "keyword": "City",
              "start": 20,
              "end": 26,
              "keywordId": 1
            },
            {
              "value": "metric",
              "keyword": "units",
              "start": 33,
              "end": 39,
              "keywordId": 3
            }
          ],
          "actions": [
            "WeatherAction"
          ],
          "creationDate": "1549387517000",
          "modificationDate": "1549387539000"
        },
        {
          "userSays": "Dublin",
          "keywords": [
            {
              "value": "Dublin",
              "keyword": "City",
              "start": 0,
              "end": 6,
              "keywordId": 1
            }
          ],
          "actions": [],
          "creationDate": "1549390456000",
          "modificationDate": "1549390490000"
        },
        {
          "userSays": "Galway",
          "keywords": [
            {
              "value": "Galway",
              "keyword": "City",
              "start": 0,
              "end": 6,
              "keywordId": 1
            }
          ],
          "actions": [],
          "creationDate": "1549390462000",
          "modificationDate": "1549390482000"
        },
        {
          "userSays": "New York",
          "keywords": [
            {
              "value": "New York",
              "keyword": "City",
              "start": 0,
              "end": 8,
              "keywordId": 1
            }
          ],
          "actions": [],
          "creationDate": "1549390464000",
          "modificationDate": "1549390470000"
        },
        {
          "userSays": "metric",
          "keywords": [
            {
              "value": "metric",
              "keyword": "units",
              "start": 0,
              "end": 6,
              "keywordId": 3
            }
          ],
          "actions": [],
          "creationDate": "1549390494000",
          "modificationDate": "1549390516000"
        },
        {
          "userSays": "imperial",
          "keywords": [
            {
              "value": "imperial",
              "keyword": "units",
              "start": 0,
              "end": 8,
              "keywordId": 3
            }
          ],
          "actions": [],
          "creationDate": "1549390496000",
          "modificationDate": "1549390504000"
        }
      ]
    }
  ],
  "keywords": [
    {
      "keywordName": "City",
      "uiColor": "#0288d1",
      "examples": [
        {
          "value": "Madrid",
          "synonyms": [
            "Madrid",
            "madrid"
          ]
        },
        {
          "value": "Barcelona",
          "synonyms": [
            "Barcelona",
            "barcelona"
          ]
        },
        {
          "value": "Seville",
          "synonyms": [
            "Seville",
            "seville"
          ]
        },
        {
          "value": "Dublin",
          "synonyms": [
            "Dublin"
          ]
        },
        {
          "value": "New York",
          "synonyms": [
            "New York"
          ]
        },
        {
          "value": "Galway",
          "synonyms": [
            "Galway"
          ]
        },
        {
          "value": "Cork",
          "synonyms": [
            "Cork"
          ]
        },
        {
          "value": "Belfast",
          "synonyms": [
            "Belfast"
          ]
        },
        {
          "value": "San Francisco",
          "synonyms": [
            "San Francisco"
          ]
        },
        {
          "value": "Chicago",
          "synonyms": [
            "Chicago"
          ]
        },
        {
          "value": "Berlin",
          "synonyms": [
            "Berlin"
          ]
        }
      ],
      "regex": "",
      "type": "learned",
      "modifiers": [],
      "creationDate": "1549372413000",
      "modificationDate": "1549386362000"
    },
    {
      "keywordName": "Cell",
      "uiColor": "#194d33",
      "examples": [
        {
          "value": "L2B_2167A",
          "synonyms": [
            "L2B_2167A"
          ]
        },
        {
          "value": "L2B_0209A",
          "synonyms": [
            "L2B_0209A"
          ]
        },
        {
          "value": "L2B_0205B",
          "synonyms": [
            "L2B_0205B"
          ]
        },
        {
          "value": "L2B_0202C",
          "synonyms": [
            "L2B_0202C"
          ]
        },
        {
          "value": "L2B_2175A",
          "synonyms": [
            "L2B_2175A"
          ]
        },
        {
          "value": "L2B_0225A",
          "synonyms": [
            "L2B_0225A"
          ]
        },
        {
          "value": "L2B_0225C",
          "synonyms": [
            "L2B_0225C"
          ]
        },
        {
          "value": "L2B_0230A",
          "synonyms": [
            "L2B_0230A"
          ]
        },
        {
          "value": "L2B_2137B",
          "synonyms": [
            "L2B_2137B"
          ]
        },
        {
          "value": "L2B_2141A",
          "synonyms": [
            "L2B_2141A"
          ]
        }
      ],
      "regex": "",
      "type": "learned",
      "modifiers": [],
      "creationDate": "1549373728000",
      "modificationDate": "1549374756000"
    },
    {
      "keywordName": "units",
      "uiColor": "#f57f17",
      "examples": [
        {
          "value": "metric",
          "synonyms": [
            "metric",
            "velcius",
            "C"
          ]
        },
        {
          "value": "imperial",
          "synonyms": [
            "imperial",
            "fahrenheit",
            "F"
          ]
        }
      ],
      "regex": "",
      "type": "learned",
      "modifiers": [],
      "creationDate": "1549386432000",
      "modificationDate": "1549386432000"
    }
  ]
}
```

**Comment by bkurowski** [comment]
Do you mind if ill send you that somewhere privately ? You email or slack ? As i have few confident data in there.

**Comment by wrathagom** [comment]
yeah PM in Slack would be great.

**Comment by dcalvom** [comment]
@wrathagom is this the scenario where two context got merged?

Please let me know to change the title of the issue. Also if this is solved let me know too in order to close the issue.

**Comment by wrathagom** [comment]
We got this on figured out here/in Slack.
